react/components/HistogramBar.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FunctionComponent } from 'react'
+import styles from '../styles.css'
+
+const createDivisions = () => {
+  let divs = []
+  for(let i = 0; i < 4; i++) {
+    divs.push(<div className={`${styles.histogramDivisions} br b--muted-4 o-50 w-100`}/>)
+  }
+  divs.push(<div className="w-100" />)
+  return divs
+}
+
+const HistogramBar: FunctionComponent<HistogramBarProps> = ({barClassName, barValueClassName, percentage, shouldShowDivisions}) => {
+  return (
+    <span className={`${styles.secondaryHistogramBar} ${barClassName} bg-white dib h1 relative v-mid ba b--muted-4 br3`}>
```

Comment:
Always use the token classes
```suggestion
    <span className={`${styles.secondaryHistogramBar} ${barClassName} bg-base dib h1 relative v-mid ba b--muted-4 br3`}>
```